packages/svelte-table/package.json

```diff
@@ -20,37 +20,40 @@
     "svelte-table",
     "datagrid"
   ],
-  "type": "commonjs",
-  "module": "build/lib/index.esm.js",
-  "main": "build/lib/index.js",
-  "types": "build/lib/index.d.ts",
+  "type": "module",
+  "module": "build/index.js",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
   "exports": {
     ".": {
-      "types": "./build/lib/index.d.ts",
-      "import": "./build/lib/index.mjs",
-      "default": "./build/lib/index.js"
+      "types": "./build/index.d.ts",
+      "import": "./build/index.js",
+      "svelte": "./build/index.js"
     },
     "./package.json": "./package.json"
   },
   "engines": {
     "node": ">=12"
   },
   "files": [
-    "build/lib/*",
-    "build/umd/*",
+    "build/*",
     "src"
   ],
   "scripts": {
     "clean": "rimraf ./build",
-    "test:types": "tsc --noEmit",
-    "build": "pnpm build:rollup && pnpm build:types",
-    "build:rollup": "rollup --config rollup.config.mjs",
-    "build:types": "tsc --emitDeclarationOnly"
+    "test:types": "svelte-check --tsconfig ./tsconfig.json",
+    "build": "svelte-package --input ./src --output ./build"
```

Comment:
since this is no longer using rollup - is the rollup config/build used anywhere else? If not, the related code can be removed.

Reply:
good call - removed the usages i found.